rbac/internal/views.py

```diff
@@ -1747,6 +1751,28 @@ def check_inventory(request):
         )
 
 
+def check_bootstrapped_tenants(request, org_id):
+    """POST to check if bootstrapped tenant is correct on relations api."""
+    tenant = get_object_or_404(Tenant, org_id=org_id)
+    if tenant and tenant.tenant_mapping:
+        default_workspace = Workspace.objects.default(tenant=tenant)
+        root_workspace = Workspace.objects.root(tenant=tenant)
+        mapping = {
+            "org_id": tenant.org_id,
+            "root_workspace": str(root_workspace.id),
+            "default_workspace": str(default_workspace.id),
+            "tenant_mapping": {
+                "default_group_uuid": str(tenant.tenant_mapping.default_group_uuid),
+                "default_admin_group_uuid": str(tenant.tenant_mapping.default_admin_group_uuid),
+                "default_role_binding_uuid": str(tenant.tenant_mapping.default_role_binding_uuid),
+                "default_admin_role_binding_uuid": str(tenant.tenant_mapping.default_admin_role_binding_uuid),
+            },
+        }
+        bootstrap_tenants_correct = BootstrappedTenantChecker._check_bootstrapped_tenants(mapping)
```

Comment:
_check_bootstrapped_tenants - "_" at the beginning is for private method, here can be  just
```
check_bootstrapped_tenants(mapping)
```